remote_states.tf

```diff
@@ -45,7 +59,7 @@ data "terraform_remote_state" "images" {
     key            = "cool-accounts/images.tfstate"
   }
 
-  workspace = "production"
+  workspace = local.workspace_type
```

Comment:
Shouldn't this be `local.workspace_name` ?
(turtles all the way down) ⬇️

Reply:
I don't think so.  `local.workspace_name` would be something like "env0-staging".  `local.workspace_type` would be "staging".  Or are you seeing something I am missing?

Reply:
It's all good.  I was misreading the code, and was confused at line 47.